english/marshmallow.md

```diff
@@ -1,8 +1,10 @@
-Your adventure begins. You realize there are hundreds of paths for you to choose from. 
-Which way should you go? Only you can decide. 
+Your adventure begins. You realize there are hundreds of paths for you to choose from.
+Which way should you go? Only you can decide.
 
 Take some [time](meditation/meditation.md) off!
 
+Are you a [girl](gender/girl.md) or a [boy](gender/boy.md)?
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.